### PR TITLE
Increase live tv buffer timeout to 30s

### PR DESCRIPTION
**Pull request by candry7731**
Update Live tv stream buffer timer so that streams do not fail while buffering. 

**Changes**
removed buffertimer duration from the xml file
added duration to the brs file to 10 seconds for all files and 30 seconds for live streams. 

**Issues**
Helps stop buffer fails on live tv streams.

### CODE REVIEW

File: components/JFVideo.brs [hunk 1]
```diff
@@ -67,8 +68,9 @@ sub ReportPlayback(state = "update" as string)
             "MediaSourceId": m.top.transcodeParams.MediaSourceId,
             "LiveStreamId": m.top.transcodeParams.LiveStreamId
         })
+        m.bufferCheckTimer.duration = 30
     end if
-
+print m.bufferCheckTimer.duration
```
Comment (jimdogx) [inline components/JFVideo.brs]: Don't forget to remove debug statements 😄

**Comment by candry7731** [comment]
Damit, I though I removed them all! I'll get better ;)

**Comment by neilsb** [comment]
Looks like something has gone wrong in the merge.  **TVShowDetails.brs** now reverts some of the changes made in a recent PR

**Review by neilsb — CHANGES_REQUESTED** [review]
There are changes to `TVShowDetails.brs` and `VideoPlayer.brs` in this PR.

If they should be included and are related to the buffer time (which they're not) can you please update the PR description to describe the changes?  If they should be included but are not related to Live TV Buffer Time can you please move them to their own PR?  Otherwise revert the changes in these files.

**Comment by candry7731** [comment]
> There are changes to `TVShowDetails.brs` and `VideoPlayer.brs` in this PR.
> 
> If they should be included and are related to the buffer time (which they're not) can you please update the PR description to describe the changes? If they should be included but are not related to Live TV Buffer Time can you please move them to their own PR? Otherwise revert the changes in these files.

This has been fixed and `TVShowDetails.brs` and `VideoPlayer.brs` have been reverted.

**Review by neilsb — APPROVED** [review]

**Pull request merged**